.github/workflows/bench-deploy.yml

```diff
@@ -56,26 +57,34 @@ jobs:
           just gpu-bench-ci fibonacci
           mv fibonacci-${{ env.COMMIT }}.json ..
         working-directory: ${{ github.workspace }}/benches
+      - uses: actions/checkout@v4
+        with:
+          repository: lurk-lab/ci-workflows
+          path: ci-workflows
       # If no plot data found, unzip all historical bench results to re-create the plots
       - name: Check for existing plot data
         run: |
-          if [ ! -f "plot-data.json" ]; then
-            shopt -s nullglob # Make glob patterns that match no files expand to a null string
-            tarballs=(./*.tar.gz)
+          shopt -s nullglob # Make glob patterns that match no files expand to a null string
+          if [ ! -f "../gh-pages/benchmarks/history/plot-data.json" ]; then
```

Comment:
Nit: would that happen to be `${{ github.workspace }}/gh-pages/...`? If so, it might be more robust to say so.